supply-chain-security/supply-chain-security-paper/sscsp.md

```diff
@@ -0,0 +1,1245 @@
+Shared with the CNCF community
+
+# Software Supply Chain Best Practices
+
+
+## Metadata
+
+| **Markdown** | | | |
+| -- | -- | -- | -- |
+| **Version** | **1.0** | 13 MAY 2021 | [Original working doc](https://docs.google.com/document/d/1VURD9rdEhiuqPdixhEozkHw01Tk6e2AaJVjBK3pK6Zc) |
+| **Status** | WIP | In Review | **Approved** |
+
+| **PDF** | |
+| -- | -- |
+| **Created** | TBD |
+| **Last Reviewed** | TBD |
+| **PDF Published** | TBD |
+| **Release Version** | 1.0 | 
+| **Final PDF Approvers** | [ ] @lizrice [ ] @justincormack  |
+
+
+| **Originating Content and Review** | |
+| -- | --|
+| **Contributors** | Andres Vega @anvega, Emily Fox @TheFoxAtWork, Faisal Razzak @faisalrazzak, @apmarshall, Cole Kennedy @colek42,Mikhail Swift @mikhailswift, Jon Meadows @jonmuk, Aditya Sirish A Yelgundhalli @adityasaky, Nisha Kumar @nishakm, Joshua Lock @joshuagl, Andrew Martin @sublimino, @mnm678 , Vinod Anandan @VinodAnandan, Magno Logan @magnologan, @rjulian @lumjjb @mlieberman85 @garrying |
+| **Reviewers** | @aspanner @mike-ensor @timfong888 @alexbarbato @voor, Alok Raj @ak-secops | 
+
+## Index
+
+-  [Executive Summary](#executive-summary)
```

Comment:
The index is flat, this is because of the multiple spaces between the bullet and the start of text i believe.